backend/translation/models.py

```diff
@@ -4,18 +4,21 @@
 
 from .metadata import LANGUAGE_CHOICES
 
+HUMAN_EDITED = "he"
+MACHINE_GENERATED = "mg"
+MANUALLY_CREATED = "mc"
+
+TRANSLATION_TYPE_CHOICES = (
+    (HUMAN_EDITED, "Human Edited"),
+    (MACHINE_GENERATED, "Machine Generated"),
+    (MANUALLY_CREATED, "Manually Created"),
+)
 
 class Translation(models.Model):
     """
     Translation model
     """
 
-    TRANSLATION_TYPE_CHOICES = [
-        ("mg", "Machine Generated"),
-        ("he", "Human Edited"),
-        ("mc", "Manually Created"),
-    ]
-
```

Comment:
Also, similar to Transcript model, shall we add an explicit `id` here, with default as `uuid4` ?

Reply:
We could but I don't see the exact benefits of doing that currently.

Reply:
Without this, the `id`'s corresponding to translations will not be UUID, right?

Reply:
Nope, they are currently integer primary keys.

Reply:
Yepp.. so what do you think? Shall we make it consistent with `transcript.id` and `video.id` ?

Reply:
yeah, that's what I was wondering. What is the need for a uuid in `translation`? Also, `video.id` is also integer id and not uuid.

Reply:
Yeah, not sure why this inconsistency came-in. I had initially suggested UUID for all primary keys instead of integer, because it is of fixed length. Hence in the future, this will give the ability for users to share their transcript or translation IDs to others and allow them to load it explicitly.

Why random UUID instead of incremental integers? So that people don't just enumerate and get others' assets.

Reply:
But yeah, I guess it's fine for now; we can come back to this later if required.

Reply:
Makes sense, I can make that change, if required. Otherwise, all other changes are done.